src/NuGet.Core/NuGet.PackageManagement/NuGet.PackageManagement.csproj

```diff
@@ -1,10 +1,17 @@
 ﻿<Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+  <!--
+    Import the SDK props manually _after_ we've imported the build customizations that
+    set things like IntermediateOutputPath, OutputPath, etc. This is required due to
+    https://github.com/Microsoft/msbuild/issues/2245. Without applying this fix,
+    a reference to netstandard.dll is never brought in, and the build fails.
+  -->
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
     <Description>NuGet Package Management</Description>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' == 'true'">netstandard2.0</TargetFrameworks>
     <NoWarn>$(NoWarn);CS1591;CS1580;CS1574;CS1573</NoWarn>
```

Comment:
Add 
   
`<TargetFramework />`

 right below the targetframeworks definition. 
It's important for the VSIX build.